doc/rbd/rbd-nomad.rst

```diff
@@ -73,38 +73,60 @@ Configure ceph-csi
 Setup Ceph Client Authentication
 --------------------------------
 
-Create a new user for nomad and `ceph-csi`. Execute the following and
-record the generated key::
+Create a new user for nomad and `ceph-csi`. Execute the following command and
+record the generated key:
+
+.. prompt:: bash $
+
+  $ ceph auth get-or-create client.nomad mon 'profile rbd' osd 'profile rbd pool=nomad' mgr 'profile rbd pool=nomad'
+
+.. code-block:: console
 
-    $ ceph auth get-or-create client.nomad mon 'profile rbd' osd 'profile rbd pool=nomad' mgr 'profile rbd pool=nomad'
     [client.nomad]
-        key = AQAlh9Rgg2vrDxAARy25T7KHabs6iskSHpAEAQ==
+      key = AQAlh9Rgg2vrDxAARy25T7KHabs6iskSHpAEAQ==
 
 
 Configure Nomad  
 ---------------
 
-By default Nomad doesn't allow containers to use privileged mode.
-Edit the nomad configuration file by adding this configuration block to `/etc/nomad.d/nomad.hcl`::
+Configuring Nomad to Allow Containers to Use Privileged Mode
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, Nomad doesn't allow containers to use privileged mode. We must
+configure Nomad so that it allows containers to use privileged mode. Edit the
+Nomad configuration file by adding the following configuration block to
+`/etc/nomad.d/nomad.hcl`::
 
     plugin "docker" {
-        config {
+      config {
         allow_privileged = true
-        }
+      }
     }
 
+Loading the rbd module
+~~~~~~~~~~~~~~~~~~~~~~
+
+Nomad must have the `rbd` module loaded. Run the following command to confirm that the `rbd` module is loaded:
+
+.. prompt:: bash $
+
+  $ lsmod |grep rbd
+
+.. code-block:: console
+
+  rbd                    94208  2
+  libceph               364544  1 rbd
 
-Nomad must have `rbd` module loaded, check if it's the case.::
+If the `rbd` module is not loaded, load it:
 
-        $ lsmod |grep rbd
-        rbd                    94208  2
-        libceph               364544  1 rbd
+.. prompt:: bash $
 
-If it's not the case, load it.::
+  sudo modprobe rbd
 
-        $ sudo modprobe rbd
+Restart Nomad
+~~~~~~~~~~~~~
 
-And restart Nomad.
+Restart Nomad.
```

Comment:
I would add a command here to justify a separate subsection ;)
```
$ systemctl restart nomad
```

Reply:
I agree with you.